internal/app/app.go

```diff
@@ -48,21 +48,11 @@ type App struct {
 	sync.Mutex
 }
 
-var (
-	lo = log.New(os.Stdout, "", log.Ldate|log.Ltime|log.Lshortfile)
+func initApp(buildString, versionString string) (fs stuffbin.FileSystem, db *sqlx.DB, ko *koanf.Koanf, lo *log.Logger) {
```

Comment:
We should replace the naked return here with an explicit return. Confusing otherwise.